README.md

```diff
@@ -224,7 +224,7 @@ If you like the simplicity of 🤗 Accelerate but would prefer a higher-level ab
 
 ## Installation
 
-This repository is tested on Python 3.7+ and PyTorch 1.4.0+
+This repository is tested on Python 3.7+ and PyTorch 1.10.0+
```

Comment:
Python version is 3.7 here. Python version PR needs to be merged first then followed by conflict resolution

Reply:
Done :)